### DISCUSSION TITLE
VERSION HARMONY LAYOUT

### DISCUSSION THREAD

**Discussion by jaimeres** [primefaces/community#705] [PrimeNG Templates] [answered]
Hello,

Why don't I have access to version 16 of the Harmony template?

![image](https://github.com/primefaces/community/assets/19354871/8ae02a42-caec-4a39-8713-c66ba7f45a2d)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It seems that there was a mistake in the server, I've just fixed it you can now access v16 of harmony-ng
